src/Packages/Passport/Runtime/Scripts/Private/Helpers/WindowsDeepLink.cs

```diff
@@ -261,7 +261,8 @@ private static string GetGameExecutablePath(string suffix)
             return Path.Combine(Application.persistentDataPath, exeName).Replace("/", "\\");
 #else
             // Returns game root directory in build
-            var exePath = Application.dataPath.Replace("/Data", "").Replace($"/{Application.productName}_Data", "");
+            var exePath = Path.Combine(Application.dataPath, "../");
```

Comment:
The extra `Path.Combine(..., "../")` seems unnecessary. Is this required to fix a specific case?

e.g. if `Application.dataPath` is `C:/ImmutableRunner/ImmutableRunner_Data`, the new version becomes `C:/ImmutableRunner/ImmutableRunner_Data/../`. Then `.Replace()` turns it into `C:/ImmutableRunner/`, which appears to be the same result as the original implementation.